dcp_prototype/frontend/src/components/layout.css

```diff
@@ -14,19 +14,19 @@ div {
 }
 
 div[role="group"][tabindex] {
-  background-color: white;
+  background-color: white !important;
```

Comment:
Could you add a comment here to flag that this is to override global css being inserted by Gatsby? If it were me I'd also either drop a #todo here to file a ticket to address this later.